pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1379,13 +1381,12 @@ func (c *PodAffinityChecker) getMatchingAntiAffinityTopologyPairsOfPods(pod *v1.
 	for _, existingPod := range existingPods {
 		existingPodNode, err := c.info.GetNodeInfo(existingPod.Spec.NodeName)
 		if err != nil {
-			if apierrors.IsNotFound(err) {
-				klog.Errorf("Pod %s has NodeName %q but node is not found",
-					podName(existingPod), existingPod.Spec.NodeName)
-				continue
-			}
 			return nil, err
 		}
+		if existingPodNode == nil {
+			klog.Infof("node %v was deleted", existingPod.Spec.NodeName)
```

Comment:
this log is unlikely to be useful, and will cause undue suspicion that something is wrong when looking at the scheduler logs.